### PR TITLE
(TWILL-238) Change TwillController restartInstances API

### PR DESCRIPTION
**Pull request by fonelor**
Implemented https://issues.apache.org/jira/browse/TWILL-238 

New method in TwillController to restart instances of runnable

### CODE REVIEW

**Comment by fonelor** [comment]
Thank you, serranom, for your comment. Is there any way to set "right" order of imports in IDEA? I've imported settings from HowToContribute page, but it seems, that it still wants to move imports.

File: twill-api/src/main/java/org/apache/twill/api/TwillController.java [hunk 1]
```diff
@@ -17,15 +17,14 @@
...
-import org.apache.twill.api.logging.LogEntry;
-import org.apache.twill.api.logging.LogHandler;
-import org.apache.twill.discovery.Discoverable;
-import org.apache.twill.discovery.ServiceDiscovered;
-
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.Future;
 import javax.annotation.Nullable;
+import org.apache.twill.api.logging.LogEntry;
```
Comment (fonelor) [inline twill-api/src/main/java/org/apache/twill/api/TwillController.java]: i've reverted reformatting changes

**Comment by fonelor** [comment]
Reverted reformatting

**Comment by chtyim** [comment]
LGTM.

**Pull request closed without merging**